### PR TITLE
Connect activity data to view.go

### PR DESCRIPTION
**Pull request by yhkaplan**

### CODE REVIEW

File: view/view.go [hunk 1]
```diff
@@ -12,10 +12,11 @@ import (
...
 	g                *gocui.Gui
-	categoryView     *gocui.View
-	categoryList     *categoryList
-	activityView     *gocui.View
-	selectedView     *gocui.View
+	categoryView     *gocui.View   // Leftmost view
+	categoryList     *categoryList // Left viewmodel
+	activityView     *gocui.View   // Rightmost view
+	activityList     *ActivityList // Right viewmodel
+	selectedView     *gocui.View   // The view currently selected by user
```
Comment (yhkaplan) [inline view/view.go:19]: Comments only here

File: view/view.go [hunk 2]
```diff
@@ -166,14 +178,22 @@ func (v *DashboardView) Run() error {
 	defaultSettings(g)
 
 	g.SetManagerFunc(v.layout)
-	//TODO: set keybindings w/ v.keybindings(g)
+	//TODO: move keybindings to separate file
+	if err := g.SetKeybinding("", gocui.KeyCtrlC, gocui.ModNone, quit); err != nil {
```
Comment (yhkaplan) [inline view/view.go:206]: You can quit now

File: view/view.go [hunk 3]
```diff
@@ -77,8 +85,11 @@ func (v *DashboardView) drawCategories() error {
 // }
 
 // New initializes the DashboardView
-func New() *DashboardView {
-	return &DashboardView{}
+func New(a []github.GitHubActivity) *DashboardView {
+	activityList := ActivityList{items: a}
+	return &DashboardView{
+		activityList: &activityList, //TODO: does this work?
```
Comment (yhkaplan) [inline view/view.go:113]: @giraffate here!

Comment (giraffate) [reply view/view.go:113]: ok:)

**Review by giraffate — APPROVED** [review]
LGTM

**Review by shimoju — APPROVED** [review]
🙆

**Pull request merged**